hphp/runtime/base/runtime-option.h

```diff
@@ -606,6 +606,12 @@ struct RuntimeOption {
   F(int64_t, PerfWarningSampleRate, 1)                                  \
   /* Profiling flags */                                                 \
   F(bool, EnableReverseDataMap, false)                                  \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, ppc64minTOCImmSize, 64)                                   \
+  /* PPC64 Option: Dump the number of elements stored in TOC */         \
+  F(bool, ppc64dumpTOCNelements, false)                                 \
+  /* PPC64 Option: Allow lwz and not only ld to be used for TOC read */ \
+  F(bool, ppc64useTOCLwz, false)                                        \
```

Comment:
We implemented these options as RuntimeOptions int order to facilitate performance tuning for each option. Do you think they should have been implemented as compilation parameters?

I think ppc64dumpTOCNelements should be kept as RuntimeOption, since it is a toggle for dumping how many values are stored in TOC.